tsdb/importer/csv/csv.go

```diff
@@ -0,0 +1,271 @@
+// Copyright 2020 The Prometheus Authors
```

Comment:
could we move this to pkg/importer instead of tsdb?

Reply:
:+1: